go-controller/pkg/ovn/policy.go

```diff
@@ -732,9 +712,9 @@ func (oc *Controller) processLocalPodSelectorSetPods(policy *knet.NetworkPolicy,
 	return
 }
 
-func (oc *Controller) processLocalPodSelectorDelPods(np *networkPolicy,
+func (oc *Controller) processLocalPodSelectorDelPods(np *networkPolicy, policy *knet.NetworkPolicy, localPods *sync.Map,
```

Comment:
@trozet shouldn't we just store the namespace/name in the networkPolicy so we don't have to pass the kube object around everywhere?

Reply:
My goal was to eliminate passing np around in the handlers, and there were so many references I had to just start removing one by one. The kapi policy we dont need any write/read protection like we do with np, so its safer. This one I couldn't get rid of the np because of:
```
// FIXME(trozet): anyway we can do this without access to the network policy object?
	// potentially re-building the NP object here? Is it worth?
	ingressDenyPorts, egressDenyPorts = oc.localPodDelDefaultDeny(np, policyPortsInfo...)
```

Will resume trying to get rid of that today.